packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayerContainer.js

```diff
@@ -3,63 +3,39 @@ import PropTypes from 'prop-types'
 import React, { Component } from 'react'
 
 import InteractionLayer from './InteractionLayer'
-import DrawingContainer from '../Drawing/DrawingContainer'
 
 function storeMapper (stores) {
-  const { addToStream } = stores.classifierStore.drawing
-  const { activeStepTasks } = stores.classifierStore.workflowSteps
+  const {
+    activeStepTasks
+  } = stores.classifierStore.workflowSteps
+  const [activeDrawingTask] = activeStepTasks.filter(task => task.type === 'drawing')
   return {
-    activeStepTasks,
-    addToStream
+    activeDrawingTask
   }
 }
 
 @inject(storeMapper)
 @observer
 class InteractionLayerContainer extends Component {
-  constructor () {
-    super()
-    this.onPointerDown = this.onPointerDown.bind(this)
-    this.onPointerMove = this.onPointerMove.bind(this)
-    this.onPointerUp = this.onPointerUp.bind(this)
-  }
-
-  onPointerDown (event) {
-    this.props.addToStream(event)
-  }
-
-  onPointerMove (event) {
-    this.props.addToStream(event)
-  }
-
-  onPointerUp (event) {
-    this.props.addToStream(event)
-  }
 
   render () {
-    const { activeStepTasks } = this.props
-    const isDrawingInActiveSteps = activeStepTasks && activeStepTasks.some(task => task.type === 'drawing')
-
+    const { activeDrawingTask, activeDrawingToolIndex, createMark, marks, svg } = this.props
```

Comment:
the `activeDrawingToolIndex, createMark, marks` props aren't being used.

Reply:
I'm going to add a `task.marks` view back to the `DrawingTask` model in #1308 so that marks disappear when you delete them.